sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -26,21 +21,68 @@ dotnet add package Azure.AI.OpenAI --prerelease
+```C# Snippet:CreateOpenAIClient
+// Replace with your Azure subscription key
+string key = "YOUR_AZURE_KEY";
+string endpoint = "https://myaccount.openai.azure.com/";
+OpenAIClient client = new OpenAIClient(new Uri(endpoint), new AzureKeyCredential(key));
+```
+
+#### Create OpenAIClient with Azure Active Directory Credential
+
+Client subscription key authentication is used in most of the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][azure_identity_dac] provider shown below,
+or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
+
+```dotnetcli
+dotnet add package Azure.Identity
+```
+
+You will also need to [register a new AAD application][aad_register_app] and [grant access][aad_grant_access] to Configuration Store by assigning the `"App Configuration Data Reader"` or `"App Configuration Data Owner"` role to your service principal.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+```C# Snippet:CreateOpenAIClientTokenCredential
+string endpoint = "https://myaccount.openai.azure.com/";
+OpenAIClient client = new OpenAIClient(new Uri(endpoint), new DefaultAzureCredential(), "myDeploymentId");
+```
 
 ## Key concepts
 
-The *Key concepts* section should describe the functionality of the main classes. Point out the most important and useful classes in the package (with links to their reference pages) and explain how those classes work together. Feel free to use bulleted lists, tables, code blocks, or even diagrams for clarity.
+The main concept to understand is [Completions][azure_openai_completions_docs]. Briefly explained, completions provides its functionality in the form of a text prompt, which by using a specific [model](https://learn.microsoft.com/azure/cognitive-services/openai/concepts/models), will then attempt to match the context and patterns, providing an output text. The following code snippet provides a rough overview (more details can be found in the `GenerateChatbotResponsesWithToken` sample code):
+
+```C#
+OpenAIClient client = new OpenAIClient(new Uri(endpoint), new DefaultAzureCredential());
+
+List<string> examplePrompts = new(){
+    //...
+    "What is Azure OpenAI?",
+    //...
+};
 
-Include the *Thread safety* and *Additional concepts* sections below at the end of your *Key concepts* section. You may remove or add links depending on what your library makes use of:
+foreach (string prompt in examplePrompts)
+{
+    CompletionsOptions request = new CompletionsOptions();
+    request.Prompt.Add(prompt);
+
+    Response<Completions> completion = client.GetCompletions("myModelDeployment", request);
```

Comment:
There was an archboard comment on this that I believe the api call here should be outside of the for loop.  The for loop is just adding the prompts to the request but that request should be sent once.

You can also use the initialization syntax and this example starts to look like this

```c#
OpenAIClient client = new OpenAIClient(new Uri(endpoint), new DefaultAzureCredential());
CompletionsOptions completionOptions = new CompletionsOptions()
{
    Prompt =
    {
        "What is Azure OpenAI?",
    }
};
Response<Completions> completion = client.GetCompletions("myModelDeployment", request);
string response = completion.Choices[0].Text;
Console.WriteLine($"Chatbot: {response}");
```

Reply:
If this is our lead-in ramp, it'd probably be good for us to use the simplified call (single prompt) in any event; multi-prompt requests already feel functionally marginal to me, and can definitely be construed as more advanced usage. At the very least, we could reserve it for a more detailed example that demonstrates the setting of other request options at the same time.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/33789

Reply:
Go for @trrwilson suggestion.